examples/tutorials/Part 11b - Secure Classification with Syft Keras and TFE: Secure Model Serving.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>"For this demo will send input data with the shape of (1, 28, 28, 1)." is not grammatical.</p><p><br></p><p>"We also return the logit instead of softmax because this operation has not yet been implemented in TF Encrypted." could use a bit more explanation but I generally get it.</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>